### DISCUSSION TITLE
Sorting my feed, can I hide upcoming videos?

### DISCUSSION THREAD

**Discussion by Paulyclees** [TeamNewPipe/NewPipe#11802] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Is there a way to hide upcoming videos in my feed? I am subscribed to a lot of channels which show videos that are upcoming later in the day to ones that are weeks away from being watched. It's nice to know something is coming soon but for convenience sake, showing only the videos that are currently accessible might be a benefit.
This might be a feature request if there is no current ability, maybe a simple flag in the settings would do or a button on the main screen to toggle upcoming on/off might be worth considering.

**Comment by asandikci** [comment]
You can do that with using fast mode.

**Comment by Stypox** [comment, marked as answer]
Yes, click on the eye button or in "Show/Hide streams" in the three-dot menu and untick "Upcoming"
